README.md

```diff
@@ -1,7 +1,51 @@
-# COMPASS
+## COMPASS
+
 COregistered Multi-temPorAl Sar Slc
 
-# License
+🚨 This toolbox is still in **pre-alpha** stage and undergoing **rapid development**. 🚨
+
+### Install
+
+
+The following will install COMPASS into a conda environment.
+
+
+1. Download source code:
+
+```bash
+git clone https://github.com/opera-adt/COMPASS.git
+```
+
+2. Install dependencies:
+
+```bash
+conda install -c conda-forge --file COMPASS/requirements.txt
+```
+
+3. Download `s1-reader` and install via pip:
+
+```bash
+git clone https://github.com/opera-adt/s1-reader.git
+python -m pip install ./s1-reader
+```
+
+4. Install `COMPASS` via pip:
```

Comment:
```suggestion
```bash
conda install -c conda-forge --file COMPASS/requirements.txt
python -m pip install git+https://github.com/opera-adt/s1-reader.git
```

3. Install `COMPASS` via pip:
```

Reply:
In the install recipe, I suggest adding `s1-reader` as part of the dependency installation.